.github/workflows/benchmark-master-push.yml

```diff
@@ -4,11 +4,6 @@ on:
   push:
     branches:
       - master
-    paths:
-      - contribs/**/*.go
```

Comment:
So if we merge a single md change, we need to run the benchmarks?

Reply:
we can make this `**/*.go`

Reply:
also, this benchmark runs in like < 1 minute now

Why do you care about the number of CI checks being executed, especially on master? Personally, I only care about the time it takes for the whole CI to run. (It's not rhetorical, it's a genuine question to understand if I'm missing something.)